docs/adr/0011-use-builder-pattern-for-model-classes.md

```diff
@@ -0,0 +1,33 @@
+# Use builder pattern for model classes
+
+Model classes should be instantiable simple without using large constructors.
+
+## Considered Alternatives
+
+* Builders
+* Setters, getters and default constructor 
+* Large constructors
+* Factories
+
+## Decision Outcome
+
+* Chosen Alternative: *Builders*
+* https://en.wikipedia.org/wiki/Builder_pattern
```

Comment:
Please move the link into the "Considered Alternatives" as markdown link: `[Builders](https://en.wikipedia.org/wiki/Builder_pattern)`